llama-index-integrations/vector_stores/llama-index-vector-stores-tablestore/llama_index/vector_stores/tablestore/base.py

```diff
@@ -16,44 +16,80 @@
     FilterCondition,
     MetadataFilter,
     FilterOperator,
+    VectorStoreQueryMode,
 )
 
 
 class TablestoreVectorStore(BasePydanticVectorStore):
-    """`Tablestore` vector store.
-
-    To use, you should have the ``tablestore`` python package installed.
+    """Tablestore vector store.
+
+    In this vector store we store the text, its embedding and
+    its metadata in Tablestore.
+
+    Args:
+        tablestore_client (OTSClient, optional): External tablestore(ots) client.
+                If this parameter is set, the following endpoint/instance_name/access_key_id/access_key_secret will be ignored.
+        endpoint (str, optional): Tablestore instance endpoint.
+        instance_name (str, optional): Tablestore instance name.
+        access_key_id (str, optional): Aliyun access key id.
+        access_key_secret (str, optional): Aliyun access key secret.
+        table_name (str, optional): Tablestore table name.
+        index_name (str, optional): Tablestore SearchIndex index name.
+        text_field (str, optional): Name of the Tablestore field that stores the text.
+        vector_field (str, optional): Name of the Tablestore field that stores the embedding.
+        ref_doc_id_field (str, optional): Name of the Tablestore field that stores the ref doc id.
+        vector_dimension (int): The dimension of the embedding vectors.
+        vector_metric_type (VectorMetricType, optional): The similarity metric type to use.
+        metadata_mappings (list[FieldSchema], optional): Custom metadata mapping is used to filter non-vector fields.
+                See the following documentation for details:
+                https://help.aliyun.com/zh/tablestore/developer-reference/create-search-indexes-by-using-python-sdk
+        kwargs (Any): Additional arguments to pass to the tablestore(ots) client.
```

Comment:
I generated the document locally, but found that the generated document contained a param in the parent class. I tried to find the reason, but couldn't find it.       If this problem can't be solved, that's okay.
![image](https://github.com/user-attachments/assets/3aa9cc1c-c252-4d8c-b603-79773412fe7e)

Reply:
That's fine. I don't think there's an easy way to exclude